test/ant/worker/generate.xsl

```diff
@@ -159,6 +159,13 @@
 						<xsl:text>Requires XSLT processor to support Java reflexive extension functions</xsl:text>
 					</xsl:when>
 
+					<xsl:when test="
+							($test-type eq 't')
```

Comment:
I'm not very familiar with this file, but I wondered why this logic doesn't apply to both `'t'` and `'s'` types.

Reply:
You're right. Fixed in ca51c236918972ecb184e33b60a178a475272f13.